lib/install/read-shrinkwrap.js

```diff
@@ -31,7 +31,7 @@ function readShrinkwrap (child, next) {
       }
       child.package._shrinkwrap = parsed
     }
-  ).then(() => next(), next)
```

Comment:
This is a functional change outside the scope of the intent of this PR.  It means that a return value will be passed to the `next` function as the first argument, which would be interpreted as an error.  Currently, no return value is being provided, but it makes the code more brittle.

Reply:
Yep, I would undo this change.